ocsp/responder.go

```diff
@@ -161,6 +166,19 @@ func NewResponder(source Source) *Responder {
 	}
 }
 
+func writeExtraHeaders(response http.ResponseWriter, headers http.Header) {
```

Comment:
this feels more like overrideHeaders, you intentionally delete original key value and add headers